src/main/views/media-account-rejection.njk

```diff
@@ -0,0 +1,141 @@
+{% from "macros/common-components.njk" import goBack, submitButton %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% extends "template.njk" %}
+{% block pageTitle %}
+ Media Account Rejection
+{% endblock %}
+{% block beforeContent %}
+  {{ super() }}
+  {{ goBack() }}
+{% endblock %}
+{% block content %}
+  <h1 class="govuk-heading-l">{{ title }}</h1>
+  {% if displayAzureError %}
+    {{ govukErrorSummary({
+      titleText: error.title,
+      errorList: [
+        {
+          text: error.azureMessage,
+          href: "#"
+        }
+      ]
+    }) }}
+  {% elseif displayRadioError %}
+    {{ govukErrorSummary({
+      titleText: error.title,
+      errorList: [
+        {
+          text: error.radioMessage,
+          href: "#"
+        }
+      ]
+    }) }}
+  {% endif %}
+  {{ govukTable({
```

Comment:
i think this should be a summary component rather than a table should it not?

Reply:
I've now switched this page (and the others over to the summary table. This also seems to have fixed the screen reader issues.